docs/platform/howto/btar/enable-backup-to-another-region.md

```diff
@@ -0,0 +1,165 @@
+
+- [Aiven Console](https://console.aiven.io/)
+- CLI
+- API
+
+### Limitations
+
+Currently, BTAR is supported for Aiven for MySQL® and Aiven for PostgreSQL® only.
+
+## Prerequisites
+
+- Aiven organization, project, and service
+- `[Pro Platform](/docs/platform/concepts/pro-platform)` enabled
+- Depending on the method you choose to use for enabling CCR
+
+  - Access to the [Aiven Console](https://console.aiven.io/)
+  - `cURL` CLI tool
+  - [Aiven CLI tool](https://github.com/aiven/aiven-client)
+
+## Back up to another region via console
+
+1. Log in to the [Aiven Console](https://console.aiven.io/).
+1. From the **Services** view, select an Aiven service on which you'd like to enable BTAR.
+1. On your service's page, select **Backups** from the sidebar.
+1. On the **Backups** page, select the actions (**...**) menu > **Secondary backup
+   location**.
+1. In the **Secondary backup location** window, use the **Secondary location** dropdown
+   menu to select a region for your additional backup. Confirm your choice by selecting
+   **Enable**.
+
+   :::tip
+   For names of the cloud regions supported in Aiven, see column *Cloud* in
+   [List of available cloud regions](/docs/platform/reference/list_of_clouds).
+   :::
+
+Your new additional backup is now visible on your service's **Backups** page in the
+**Secondary backup location** column.
+
+## Back up to another region with CLI
+
+:::note
+In this instruction, the [Aiven CLI client](/docs/tools/cli) is used to interact
+with Aiven APIs.
+:::
+
+Using CLI, you can enable BTAR for
+
+- [New Aiven service](#new-service-cli) or
+- [Existing Aiven service](#existing-service-cli)
+
+:::note[additional_backup_regions]
+To enable BTAR on an Aiven service, you need to add the ``additional_backup_regions``
+parameter to relevant commands.
+:::
+
+### Create a new service with BTAR via CLI {#new-service-cli}
+
+Use the [avn service create](/docs/tools/cli/service-cli) command to create a new
+service. Include ``additional_backup_regions`` as a parameter to the command and set its
+value to the name of desired cloud region.
+
+```bash
+    avn service create                                      \
+        --service-type service_type_name                    \
+        --cloud cloud_region_name                           \
+        --plan service_plan_name                            \
+        -c additional_backup_regions=[name_of_cloud_region] \
+        new_service_name
+```
+
+### Enable BTAR on an existing service via CLI {#existing-service-cli}
+
+Use the [avn service update](/docs/tools/cli/service-cli) command to configure your
+service so that it supports BTAR. Include ``additional_backup_regions`` as a parameter to
+the command and set its value to the name of desired cloud region.
+
+```bash
+    avn service update name_of_existing_service                    \
+        -c additional_backup_regions=[\"name_of_cloud_region\"]
```

Comment:
enabling BTAR on an existing service should be working fine, but modifying it will be rate limited (so user cannot change it very often) and will trigger a new base backup. both of these still need to be implemented [before going GA], but probably it's worth mentioning somewhere.